packages/react/src/themes/teams/components/Chat/chatMessageVariables.ts

```diff
@@ -15,7 +15,7 @@ export interface ChatMessageVariables {
   authorFontWeight: number
   headerMarginBottom: string
   contentColor: string
-  contentFocusOutlineColor: string
+  linkColor: string
```

Comment:
The links on the mineMessages are a bit different shade then the other messages. Should we introduce variable for those variations?

Reply:
added `linkColorMine` variable as discussed